.env.test

```diff
@@ -4,7 +4,7 @@
 GETH_DOCKER_IMAGE=ethereum/client-go:v1.13.14
 # Use either image or local binary for the testing
 GETH_BINARY_DIR=
-LIGHTHOUSE_DOCKER_IMAGE=sigp/lighthouse:latest-amd64-unstable
+LIGHTHOUSE_DOCKER_IMAGE=sigp/lighthouse:v7.0.1
```

Comment:
Don't we want to do `v7.0.1-amd64`?

Reply:
their image is multi arch, I am not exactly sure why we specifically used `amd64` but this allows to run sim tests locally on any platform